ci/experimental/build_docker.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -e
+#
+# Build Snowflake Python Connector in Docker
+# NOTES:
+#   - To compile only a specific version(s) pass in versions like: `./build_docker.sh "3.5 3.6"`
+set -o pipefail
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source $THIS_DIR/set_base_image.sh
+CONNECTOR_DIR="$(dirname "$( dirname "${THIS_DIR}")")"
```

Comment:
Just looks a bit neater when the constants are all defined at the beginning in 1 block
```suggestion
CONNECTOR_DIR="$(dirname "$( dirname "${THIS_DIR}")")"
source $THIS_DIR/set_base_image.sh
```